### ISSUE TITLE
Lang folder subfolders not supported

### ISSUE THREAD

**Issue by LukeSavefrogs** [LarsWiegers/laravel-translations-checker#56]
My lang folder currently has the following nested structure:
```
📦lang
 ┣ 📂en
 ┃ ┣ 📂filament
 ┃ ┃ ┣ 📜pages.php
 ┃ ┃ ┣ 📜resources.php
 ┃ ┃ ┗ 📜widgets.php
 ┃ ┣ 📜auth.php
 ┃ ┣ 📜pagination.php
 ┃ ┣ 📜passwords.php
 ┃ ┗ 📜validation.php
 ┣ 📂it
 ┃ ┣ 📂filament
 ┃ ┃ ┣ 📜pages.php
 ┃ ┃ ┣ 📜resources.php
 ┃ ┃ ┗ 📜widgets.php
 ┃ ┣ 📜auth.php
 ┃ ┣ 📜pagination.php
 ┃ ┣ 📜passwords.php
 ┃ ┗ 📜validation.php
```

The package however raises false positives since does not handle subfolders in the lang folder (e.g. `en/filament/widgets.php` instead of `en/widgets.php`):
```
$ sail artisan translations:check --excludedDirectories=vendor --excludedFileExtensions=json
The language en (/var/www/html/lang/en) is missing the file ( widgets.php )
The language it (/var/www/html/lang/it) is missing the file ( widgets.php )
The language en (/var/www/html/lang/en) is missing the file ( pages.php )
The language it (/var/www/html/lang/it) is missing the file ( pages.php )
The language en (/var/www/html/lang/en) is missing the file ( resources.php )
The language it (/var/www/html/lang/it) is missing the file ( resources.php )
[ ... ]
Missing the translation with key: en.resources.mailables.remember_to_return
Missing the translation with key: it.resources.mailables.remember_to_return
```

A quick workaround would be to replace organizing in subfolders with using a dash in the filename (e.g. `en/filament/widgets.php`  becomes `en/filament-widgets.php` ) but it limits the flexibility and it forces someone to adapt the repository structure to the package limits.

**Comment by M-Zoldak** [comment]
I've just created PR(https://github.com/LarsWiegers/laravel-translations-checker/pull/57) that fixes this behavior.
